core/src/main/java/org/apache/carbondata/core/scan/complextypes/ArrayQueryType.java

```diff
@@ -97,4 +99,56 @@ public void parseBlocksAndReturnComplexColumnByteArray(DimensionRawColumnChunk[]
     return DataTypeUtil.getDataTypeConverter().wrapWithGenericArrayData(data);
   }
 
+  @Override public Object getDataBasedOnColumn(ByteBuffer dataBuffer, CarbonDimension parent,
```

Comment:
Don't implement it, throw unsupported exception

Reply:
Okay.done